data/receipt/receipt_test.go

```diff
@@ -13,7 +13,7 @@ func TestReceipt_SettersAndGetters(t *testing.T) {
 	t.Parallel()
 
 	r := receipt.Receipt{}
-	assert.False(t, r.IsInterfaceNil())
+	assert.False(t, check.IfNil(&r))
```

Comment:
useless assert as r is value not pointer

Reply:
removed